data/jakarta/utils.sh

```diff
@@ -126,3 +126,97 @@ snap_remove()
     snap remove edgexfoundry || true
 }
 
+# wait for services to come online
+# NOTE: this may have to be significantly increased on arm64 or low RAM platforms
+# to accomodate time for everything to come online
+snap_wait_all_services_online()
+{
+    regex="4[0-9][0-9]|5[0-9][0-9]"
```

Comment:
Choose a better name for this variable. E.g. `http_errcode_regex`

Reply:
Thanks! Updated.